README.md

```diff
@@ -68,9 +68,12 @@ Additional build and runtime packages to be installed can be listed in `args`,
 under the `BUILD_PACKAGES` and `RUNTIME_PACKAGES` keys, respectively. It is not
 necessary to quote them - e.g. `BUILD_PACKAGES: python3 python3-requests`.
 Packages that strictly need to be installed via `pip` can be listed under the
-`RUNTIME_PIP_PACKAGES` key.
-Packages essential to all (or most) IOCs should be added to
-[this repository's `Dockerfile`](./Dockerfile).
+`RUNTIME_PIP_PACKAGES` key. Extra files can be downloaded and installed to the
+container by listing their TAR or ZIP archive URLs under `BUILD_TAR_PACKAGES`
+or `RUNTIME_TAR_PACKAGES`. To use HTTPS (or any other TLS-based protocol),
+`ca-certificates` must also be added in `RUNTIME_PACKAGES`. Packages essential
+to all (or most) IOCs should be added to [this repository's
+`Dockerfile`](./Dockerfile).
```

Comment:
I would split the new content into its own paragraph. And keep the "Packages essential..." part where it used to be.

Reply:
> And keep the "Packages essential..." part where it used to be.

Do you mean right after pip install of things?